server/src/controllers/temp-outfitter.es6

```diff
@@ -37,21 +33,17 @@ const translateFromClientToDatabase = (input, output) => {
   output.applicantInfoDayPhonePrefix = input.applicantInfo.dayPhone.prefix;
   output.applicantInfoEmailAddress = input.applicantInfo.emailAddress;
   output.applicantInfoEveningPhoneAreaCode =
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.areaCode
-      ? input.applicantInfo.eveningPhone.areaCode
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.areaCode ? input.applicantInfo.eveningPhone.areaCode :
+      null;
   output.applicantInfoEveningPhoneExtension =
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.extension
-      ? input.applicantInfo.eveningPhone.extension
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.extension ? input.applicantInfo.eveningPhone
+      .extension : null;
   output.applicantInfoEveningPhoneNumber =
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.number
-      ? input.applicantInfo.eveningPhone.number
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.number ? input.applicantInfo.eveningPhone.number :
```

Comment:
this seems like it exceeds the linting line length?

Reply:
@lauraGgit there's no linting limit on the server code, so I am configuring it to be 125. Frontend has  200 characters limit.

Reply:
linting should conform to 18f's style guide which is based off of `airbnb`'s https://github.com/airbnb/javascript> they say 100 actually, but given how late we are in 125 should be ok https://github.com/airbnb/javascript#whitespace--max-len